Steamworks/src/org/usfirst/frc/team4786/robot/Robot.java

```diff
@@ -100,6 +100,8 @@ public void teleopInit() {
 	@Override
 	public void teleopPeriodic() {
 		Scheduler.getInstance().run();
+		SmartDashboard.putBoolean("Gear Present", Gear.gearLimitSwitchPressed());
```

Comment:
This is fine now, but we eventually will want to remove this. We will eventually make a smart dashboard subsystem that handles everything in a default command.